spirv_msl.cpp

```diff
@@ -8373,6 +8378,95 @@ void CompilerMSL::emit_instruction(const Instruction &instruction)
 			SPIRV_CROSS_THROW("Raster order groups require MSL 2.0.");
 		break; // Nothing to do in the body
 
+	case OpRayQueryInitializeKHR:
+	{
+		flush_variable_declaration(ops[0]);
+
+		statement("intersection_params _intersection_params_;");
```

Comment:
What does this do? Also, there will be conflicts with naming if multiple such variables are declared in same scope.

Reply:
Fixed.